sdk/core/azure-core/inc/azure/core/datetime.hpp

```diff
@@ -115,7 +115,6 @@ class DateTime final : public _detail::Clock::time_point {
 
   /**
    * @brief Construct an instance of #Azure::DateTime from base class.
-   *
```

Comment:
Revert changes to these files. I believe @vhvb1989 added them intentionally to fix doxygen docs.

Reply:
Thanks - no idea where they came from. One would be ok, but all 3... I don't rebember touching them. Maybe was some bad merge.